src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.ts

```diff
@@ -163,4 +180,18 @@ export class DashboardV3Component extends PrometheusListHelper implements OnInit
   trackByFn(index: any) {
     return index;
   }
+
+  getHardwareConfig(): Observable<any> {
+    return this.mgrModuleService.getConfig('cephadm').pipe(
+      switchMap((resp: any) => {
+        this.hardwareEnabled = resp?.hw_monitoring;
+        return of(null);
+      })
+    );
+  }
+
+  setHardwareError(error: boolean): void {
+    this.hardwareError = error;
+    this.cdRef.detectChanges();
```

Comment:
we can remove this and especially the manual change detection since that can cause other issues. let's not do this and store the hardware error like i did in the above comment

Reply:
Removed it, thanks.